res/controllers/Hercules-DJControl-Inpulse-300-script.js

```diff
@@ -3,7 +3,7 @@
 // ***************************************************************************
 // * Mixxx mapping script file for the Hercules DJControl Inpulse 300.
 // * Author: DJ Phatso, contributions by Kerrick Staley and BoredGuy1
-// * Version 1.3 (Feb 2024)
```

Comment:
Makes sense to update the version, too, no?
(this is no review, just stumbled over this)

Reply:
Interesting. It's up-to-date on my end. This is what I see:
![image](https://github.com/mixxxdj/mixxx/assets/66702733/d3fd3958-b39a-4140-bc6a-c0bd0ad05402)

I will try to iron out the formatting issues though. Turns out I didn't have ESLint installed, so all the formatting checks were automatically being passed. Oops...